### PR TITLE
switch to using gamertags for connected storage

### PR DESCRIPTION
**Pull request by brianwp3000**
Titlestorage recently added support for using gamertags instead of xuids in requests. This PR changes the tool to use those gamertag APIs, which makes it easier for developers to take advantage of the tool (since they often don't have the xuids of their users).

TitleStorage does not accept modern gamertags--it only accepts legacy ones (the ones without a #XXX at the end). I tried to be clear about that in the docs, but I do not know if "modern" and "legacy" is the correct terminology, nor do I know if there are resources for developers convert modern gamertags to legacy ones.

### CODE REVIEW

**Comment by brianwp3000** [comment]
@rakhad for review

File: CommandLine/XblConnectedStorage/Program.cs [hunk 1]
```diff
@@ -78,11 +84,11 @@ private static async Task<int> Main(string[] args)
 
         private static async Task<int> OnDownload(Options options)
         {
-            ulong xuid = options.XboxUserId;
+            string gamertag = options.GamerTag;
```
Comment (jasonsandlin) [inline CommandLine/XblConnectedStorage/Program.cs:87]: make sure spaces work.  aka "2 Dev 1234" which are comment test account names.

Comment (brianwp3000) [reply CommandLine/XblConnectedStorage/Program.cs:87]: Tested and working

**Review by rakhad — APPROVED** [review]
:shipit:

**Pull request merged**